code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -318,15 +318,15 @@
 
 /datum/reagent/medicine/omnizine
 	name = "Omnizine"
-	description = "Slowly heals all damage types. Overdose will cause damage in all types instead."
+	description = "Heals all damage types, but is rapidly metabolized. Overdose will cause damage in all types instead."
 	reagent_state = LIQUID
 	color = "#DCDCDC"
-	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	metabolization_rate = 2
```

Comment:
why not 2 * REAGENTS_METABOLISM

Reply:
2 * REAGENTS_METABOLISM != 2

Reply:
what is it

Reply:
0.8, in /tg/

Reply:
Dude, 2u/tick is INSANELY fast. Under your changes, a single 5u dose of omnizine from the CMO's hypospray will heal a mere 1.2 points of toxin and oxygen damage and 1.5 points of brute and burn damage. That is PATHETIC. Vaping omnizine will become the only viable way to heal from the chem.

Also, if you want to make the metabolization rate of omnizine 2u/tick, make it equal to 5 * REAGENTS_METABOLISM (which is still 2u/tick, but now omnizine's metabolization rate will be changed if REAGENTS_METABOLISM is change (just like how the metabolization rates of almost all other chems would be changed if that were to happen)).